web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -168,8 +168,8 @@
 import static datawave.webservice.query.cache.QueryTraceCache.PatternWrapper;
 
 @Path("/Query")
-@RolesAllowed({"AuthorizedUser", "AuthorizedQueryServer", "PrivilegedUser", "InternalUser", "Administrator", "JBossAdministrator"})
-@DeclareRoles({"AuthorizedUser", "AuthorizedQueryServer", "PrivilegedUser", "InternalUser", "Administrator", "JBossAdministrator"})
+@RolesAllowed({"AuthorizedUser", "AuthorizedQueryServer", "PrivilegedUser", "UnlimitedQueryResultsUser", "InternalUser", "Administrator", "JBossAdministrator"})
+@DeclareRoles({"AuthorizedUser", "AuthorizedQueryServer", "PrivilegedUser", "UnlimitedQueryResultsUser", "InternalUser", "Administrator", "JBossAdministrator"})
```

Comment:
Do we really want to add the UnlimitedQueryResultsUser role here?  This is an OR'd list of users allowed to call this method.  By doing this, a user wouldn't have to be an AuthorizedUser to run queries.

Reply:
Are PrivilegedUser/UnlimitedQueryResultsUser permutations of the AuthorizedUser by default? If it doesn't need to be added to the RolesAllowed would that also mean that PrivilegedUser could be removed from this list as well?

Reply:
I don't think it's possible to have a PrivilegedUser that is not an AuthorizedUser, so PrivilegedUser probably doesn't need to be in this list.  The same probably goes for Administrator and JBossAdministrator to be honest.

Reply:
Got it. Updated the beans to remove the extra roles